app/views/idv/doc_auth/welcome.html.erb

```diff
@@ -117,10 +103,4 @@
     </div>
   <% end %>
 
-
-  <%= javascript_packs_tag_once(
-    'clipboard',
-    'ial2-consent-button',
-    'document-capture-welcome',
```

Comment:
This script adds the `no_camera` input that the `WelcomeStep` class uses to determine if the user should be redirected to the `idv_doc_auth_errors_no_camera_url` error screen. Do we need to move that step code into the new step? Or conversely, keep the script being loaded here?

Reply:
Nice catch!